chia/wallet/wallet_state_manager.py

```diff
@@ -692,6 +712,127 @@ async def determine_coin_type(
 
         return None
 
+    def deserialize_coin_metadata(self, metadata: Optional[VersionedBlob], coin_type: CoinType) -> Any:
+        if metadata is None:
+            return None
+
+        if coin_type == CoinType.CLAWBACK:
+            return ClawbackMetadata.from_bytes(metadata.blob).to_json_dict()
+        else:
+            return metadata.blob
+
+    async def auto_claim_coins(self) -> None:
+        # Get unspent clawback coin
+        unspent_coins = await self.coin_store.get_all_unspent_coins(coin_type=CoinType.CLAWBACK)
+        current_timestamp = self.blockchain.get_latest_timestamp()
+        clawback_coins: Dict[Coin, ClawbackMetadata] = {}
+        tx_fee = uint64(self.config.get("auto_claim", {}).get("tx_fee", 0))
+        min_amount = uint64(self.config.get("auto_claim", {}).get("min_amount", 0))
+        for coin in unspent_coins:
+            if coin.metadata is None or coin.metadata.blob is None or len(coin.metadata.blob) <= 1:
```

Comment:
Like mentioned in the other place (i think it was RPC) this should probably be an `assert` instead and keep the try/except and log/continue thing in mind in case you think its needed and you apply it there too.
```suggestion
            assert coin.metadata is not None, f"metadata is None for {coin.name().hex()}"
```

Reply:
Fixed